cli/src/main/java/com/devonfw/tools/ide/tool/npm/NpmBasedCommandlet.java

```diff
@@ -54,7 +54,8 @@ protected VersionIdentifier runPackageManagerGetInstalledVersion(String npmPacka
       this.context.trace("Since node is not installed, also package {} for tool {} cannot be installed.", npmPackage, this.tool);
       return null;
     }
-    ProcessResult result = runPackageManager(ProcessMode.DEFAULT_CAPTURE, ProcessErrorHandling.NONE, "list", "-g", npmPackage, "--depth=0");
+    ProcessResult result = runPackageManager(ProcessMode.DEFAULT_CAPTURE, ProcessErrorHandling.NONE, "list", "-g", npmPackage, "--depth=0", "--prefix",
```

Comment:
If we always need to add `--prefix` you should add it centrally in `runPackageManager` to avoid redundancies.

Reply:
I've removed the redundancies now and added a new StringUtil method for this case.